packages/graphback-core/src/crud/mappingHelpers.ts

```diff
@@ -157,17 +156,24 @@ export function getInputFieldName(field: GraphQLField<any, any>): string {
   return relationshipAnnotation.key || transformForeignKeyName(field.name);
 }
 
-export function getInputFieldNamedType(schemaComposer: SchemaComposer<any>, field: GraphQLField<any, any>, operation: GraphbackOperationType): GraphQLInputType {
+export function getInputFieldNamedTypeName(modelName: string, field: GraphQLField<any, any>, operation: GraphbackOperationType): string {
```

Comment:
Probably no need for the NamedType part anymore.

```suggestion
export function getInputFieldTypeName(modelName: string, field: GraphQLField<any, any>, operation: GraphbackOperationType): string {
```

Reply:
Good catch!